src/Modal.js

```diff
@@ -177,7 +177,7 @@ class ModalManager {
 
         var modal = this._modals[0];
         var dialog = (
-            <div className={"mx_Dialog_wrapper " + modal.className}>
+            <div className={"mx_Dialog_wrapper " + (modal.className ? modal.className : '') }>
```

Comment:
we do. but it's arguably slightly overkill for something trivial like this. plus this was just fixing someone else's bug.